src/main/java/com/example/bookshop/controller/BookController.java

```diff
@@ -19,46 +22,55 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/books")
 public class BookController {
     private final BookService bookService;
 
+    @Operation(summary = "find all books where authors with certain name",
+                description = "find all books by athours")
     @GetMapping("/author")
     public List<BookDto> findAllByAuthor(@RequestParam String author) {
         return bookService.findAllByAuthor(author);
     }
 
+    @Operation(summary = "Create a new book", description = "add information to db")
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
     public BookDto create(@RequestBody @Valid CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
 
+    @Operation(summary = "Get all books", description = "amount can be limited by parameter")
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.getAll();
+    public List<BookDto> getAll(Pageable pageable) {

```

Comment:
```suggestion
    public List<BookDto> getAll(@PageableDefault Pageable pageable) {

```
It is better to specify @PageableDefault . If the page is not specified in the request, the default value will be used (otherwise an error will occur)